sample.tf

```diff
@@ -9,7 +9,7 @@ terraform {
 }
 
 provider "artifactory" {
-  //  supply ARTIFACTORY_USERNAME, _PASSWORD and _URL as env vars
+  //  supply ARTIFACTORY_ACCESS_TOKEN / _API_KEY and ARTIFACTORY_URL as env vars
```

Comment:
Let's have the full env var `ARTIFACTORY_API_KEY` here.

Reply:
ok